akka-docs/rst/scala/remoting.rst

```diff
@@ -627,3 +627,5 @@ special configuration that sets both the logical and the bind pairs for remoting
       }
    }
   }
+ 
+Keep in mind that local.address must be in one of following ranges: *172.16.xxx.xxx*, *172.31.xxx.xxx*, *192.168.xxx.xxx*, *10.xxx.xxx.xxx*.
```

Comment:
Hmm... Can some docker expert confirm if this is a good hint to give? Is it always those ranges? Isn't there a docker command which would list the  usable addresses?